integration-test/src/test/java/org/uniprot/store/indexer/search/uniprot/FullCISplitNumericAnalysisSearchIT.java

```diff
@@ -709,6 +709,23 @@ void canFindExactPhraseWithDash3(FullCISplitNumericAnalysisSearchIT.FieldType fi
                 .canBeFound(field);
     }
 
+    @ParameterizedTest
+    @EnumSource(FullCISplitNumericAnalysisSearchIT.FieldType.class)
+    void canFindQueryEndingWithNumber(FullCISplitNumericAnalysisSearchIT.FieldType field) {
+        String accession = newAccession();
+
+        String indexFieldValue =
+                "Human immunodeficiency virus type 1 group M subtype B (isolate BH10) (HIV-1)";
+        String queryFieldValue = "Human immunodeficiency virus 1";
+        String query = fieldQuery(field.getQueryField(), queryFieldValue);
+
```

Comment:
To mimic the user's query, could we please add a search string without space between alphabet and number. e.g. Axm1